app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallViewModel.kt

```diff
@@ -103,4 +139,14 @@ class OngoingCallViewModel @Inject constructor(
             }
         }
     }
+
+    override fun onCleared() {
+        uiScope.launch {
+            if (callEstablishedState.isCameraOn)
+                pauseVideo()
+            onCleared()
+            viewModelJob.cancel()
+        }
```

Comment:
this is done when using viewModelScope extension, jobs get cancelled on the onCleared callback, I think it is safe to relay on viewModelScope instead of creating custom scope, unless I do not get something